packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './xdsl-meetings.controller';
+import template from './xdsl-meetings.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    serviceName: '<',
+    loadMeetings: '<',
```

Comment:
Seems not used in the controller/template, so no need to expose it as a binding (not all resolves are bindings)